ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -247,6 +382,14 @@ KEXT_STATIC int HandleVnodeOperation(
     char procname[MAXCOMLEN + 1] = "";
     bool isDeleteAction = false;
     bool isDirectory = false;
+    bool isRename = true;
+
+    isDeleteAction = ActionBitIsSet(action, KAUTH_VNODE_DELETE);
+    if (isDeleteAction && VnodeIsEligibleForEventHandling(currentVnode))
+    {
+        isRename = DeleteOpIsForRename(currentVnode);
+        KextLog_FileInfo(currentVnode, "HandleVnodeOperation thread = %p, vnode = %p: KAUTH_VNODE_DELETE: isRename = %s", current_thread(), currentVnode, isRename ? "YES" : "NO");
```

Comment:
Debugging info?

Reply:
Yep, looks like that one made it past my removal pass.